packages/mdl-fab/mdl-fab.scss

```diff
@@ -0,0 +1,106 @@
+/**
+ * Copyright 2016 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+@import "variables";
+
+.mdl-fab {
+  @include mdl-shadow-4dp;
+  display: inline-block;
+  position: relative;
+  width: 56px;
+  height: 56px;
+  padding: 0;
+  border: none;
+  border-radius: 50%;
+  background-color: var(--mdl-accent-color);
+  color: white;
+  text-align: center;
+  cursor: pointer;
+  overflow: hidden;
+  user-select: none;
+  box-sizing: border-box;
+  -webkit-appearance: none;
+
+  &--mini {
+    width: 40px;
+    height: 40px;
+  }
+
+  &--plain {
+    background-color: white;
+    color: rgba(0, 0, 0, .87);
```

Comment:
rather than hard-coding this, we should specify `mdl-typography` as a dependency and then:

```scss
@import "mdl-typography/mixins";  // won't output any additional css

// ...
&--plain {
  background-color: white;
  color: mdl-typography-color(primary, /* $dark: */ true);
}
```

This makes our code a bit more maintainable.